src/protocols/secure_channel/BUILD.gn

```diff
@@ -29,6 +29,8 @@ static_library("secure_channel") {
     "CASEServer.h",
     "CASESession.cpp",
     "CASESession.h",
+    "CheckinMessage.cpp",
```

Comment:
This is using `crypto`, so should probably add that to `public_deps` (since it's using it in the API).

Reply:
done